README.markdown

```diff
@@ -191,9 +197,21 @@ We plan to provide first class `async` support in a future release.
 
 If a job raises an exception, it is logged and handed off to the
 `Resque::Failure` module. Failures are logged either locally in Redis
-or using some different backend.
+or using some different backend. To see exceptions while developing,
+see details below under Logging.
 
-For example, Resque ships with Hoptoad support.
+For example, Resque ships with Airbrake support. To configure it, put
+the following into an initialisation file or into your rake job:
+
+``` ruby
+# send errors which occur in background jobs to redis and airbrake
+require 'resque/failure/multiple'
+require 'resque/failure/redis'
+require 'resque/failure/airbrake'
+
+Resque::Failure::Multiple.classes = [Resque::Failure::Redis, Resque::Failure::Airbrake]
+Resque::Failure.backend = Resque::Failure::Multiple
```

Comment:
TODO: is there Github code that this feature would clean up? Haystack integration?